it/test/connectors/OttConnectorSpec.scala

```diff
@@ -80,4 +82,114 @@ class OttConnectorSpec
+                |            "id": "238dbab8cc5026c67757c7e05751f312",
+                |            "type": "category_assessment"
+                |          }
+                |        ]
+                |      }
+                |    }
+                |  },
+                |  "included": [
+                |    {
+                |      "id": "238dbab8cc5026c67757c7e05751f312",
+                |      "type": "category_assessment",
+                |      "relationships": {
+                |        "exemptions": {
+                |          "data": [
+                |            {
+                |              "id": "8392",
+                |              "type": "additional_code"
+                |            }
+                |          ]
+                |        },
+                |        "theme": {
+                |          "data": {
+                |            "id": "1.1",
+                |            "type": "theme"
+                |          }
+                |        },
+                |        "geographical_area": {
+                |          "data": {
+                |            "id": "IQ",
+                |            "type": "geographical_area"
+                |          }
+                |        },
+                |        "excluded_geographical_areas": {
+                |          "data": [
+                |
+                |          ]
+                |        },
+                |        "measure_type": {
+                |          "data": {
+                |            "id": "465",
+                |            "type": "measure_type"
+                |          }
+                |        },
+                |        "regulation": {
+                |          "data": {
+                |            "id": "R0312100",
+                |            "type": "legal_act"
+                |          }
+                |        },
+                |        "measures": {
+                |          "data": [
+                |            {
+                |              "id": "2524368",
+                |              "type": "measure"
+                |            }
+                |          ]
+                |        }
+                |      }
+                |    }
+                |  ]
+                |}""".stripMargin
+            )
+          )
+      )
+
+      val connectorResponse = connector.getCategorisationInfo("123456").futureValue
+      connectorResponse.categoryAssessments.size mustEqual 1
+      connectorResponse.categoryAssessments(0).id mustEqual "238dbab8cc5026c67757c7e05751f312"
+    }
+
+    "must return a failed future when the server returns an error" in {
+
+      wireMockServer.stubFor(
+        get(urlEqualTo(s"/ott/goods-nomenclatures/123456"))
+          .willReturn(serverError())
+      )
+
+      connector.getCategorisationInfo("123456").failed.futureValue
+    }
+
+    "must return a not found future when the server returns a not found" in {
```

Comment:
this is just returning a failed future, not a "not found future"

Reply:
Done :)